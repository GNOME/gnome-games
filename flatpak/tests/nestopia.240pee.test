[Retro Reftest]
Path=/nestopia/240pSuite
Core=/app/lib/libretro/nestopia_libretro.so
Medias=240pee.nes;

[Frame 120]
Video=nestopia.240pee.120.png
