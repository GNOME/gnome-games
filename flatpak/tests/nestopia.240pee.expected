/nestopia/240pSuite/Boot: OK
/nestopia/240pSuite/Options: OK
/nestopia/240pSuite/120/FastForward: OK
/nestopia/240pSuite/120/Run: OK
/nestopia/240pSuite/120/Video: OK
